src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -349,20 +350,67 @@ private void AbortWorkflow(IProgressController controller, CancellationToken tok
             return this.BindingProjects.Select(Language.ForProject).Distinct();
         }
 
-        private void InformAboutFilteredOutProjects(IEnumerable<Project> filteredOut)
+        private void InformAboutFilteredOutProjects()
         {
-            List<Project> projects = filteredOut.ToList();
-            if (projects.Count == 0)
+            var includedProjects = this.BindingProjects.ToList();
+            var excludedProjects = this.projectSystem.GetSolutionProjects().Except(this.BindingProjects).ToList();
+
+            var output = new StringBuilder();
+
+            output.AppendFormat(Strings.SubTextPaddingFormat, Strings.DiscoveringSolutionIncludedProjectsHeader).AppendLine();
```

Comment:
There seems to be some duplication here with the way we print the included and excluded projects.